treeherder/etl/perf.py

```diff
@@ -333,12 +333,54 @@ def _load_perf_datum(job: Job, perf_datum: dict):
                 generate_alerts.apply_async(args=[signature.id], queue="generate_perf_alerts")
 
 
+def _is_suite_allowed(suites: list, framework_name: str) -> bool:
+    allowlist_frameworks_suites = {
+        "awsy": ["ALL"],
+        "build_metrics": ["decision", "compiler warnings"],
+        "browsertime": ["constant-regression"],
+    }
```

Comment:
The job is processed if at least one suite name matches the allowlist (e.g. `compiler warnings`).

[0]
https://firefoxci.taskcluster-artifacts.net/KZ6krBACTcyC1_q_tUejTA/0/public/build/perfherder-data-building.json